pkg/integrations/v2/app_o11y_receiver/models/meta.go

```diff
@@ -0,0 +1,150 @@
+package models
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/grafana/agent/pkg/integrations/v2/app_o11y_receiver/utils"
+)
+
+// Sdk holds metadata about the app agent that produced the event
+type Sdk struct {
```

Comment:
nit: this should be renamed to SDK to match with go naming conventions for exported acronyms.

Reply:
dang, I knew this! 
- [ ] capitalize